modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-lvmcluster_{context}"]
+= About the LVMCluster custom resource
+
+After you have installed {lvms}, you must create an `LVMCluster` custom resource (CR) on a worker node.
+
+You can configure the `LVMCluster` CR to perform the following actions:
+
+* Create LVM volume groups that you can use to provision persistent volume claims (PVCs).
+* Configure a list of devices that you want to add to the LVM volume groups. 
+* Configure the requirements to select the nodes on which you want to create an LVM volume group, and the thin pool configuration for the volume group.
+* Force wipe the selected devices.
+
+You can create an `LVMCluster` CR using the OpenShift CLI (`oc`) or the {product-title} web console.
+
+include::snippets/lvms-creating-lvmcluster.adoc[]
+
+The `LVMCluster` CR contains the following fields:
+
+`deviceClasses`:: Contains the configuration to assign the local storage devices to the LVM volume groups. 
++
+To create multiple device storage classes, create a device class for each storage class. 
++
+After creating an `LVMCluster` CR, if you add or remove a device class, the update reflects in the `LVMCluster` CR only after deleting and recreating the `TopoLVM-Node` pod. 
```

Comment:
This only goes up to 4.15, 4.16 and above do not have this limitation, maybe worth a separate docs issue?

Reply:
Yes. It's best to have a separate doc issue for this in 4.16.